posts/2021/02/accessibility-whos-responsible.md

```diff
@@ -0,0 +1,130 @@
+<!--
+.. title: Accessibility: Who's Responsible?
+.. slug: accessibility-whos-responsible
+.. date: 2021-02-28 08:00:00 UTC-00:00
+.. author: Isabela Presedo-Floyd
+.. tags: JupyterLab, Accessibility, JLabA11y
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+## JupyterLab Accessibility Journey Part 1
+
+For the past few months I've been part of a group of people in the JupyterLab community 
+who've committed to start chipping away at the many accessibility failings of JupyterLab. 
+I find this work is critical, fascinating, and a learning experience for everyone involved, 
+so I'm going to document my personal experience and lessons I've learned in a series of blog 
+posts. Welcome!
+
+<!-- TEASER_END -->
+
+Because this is the first of a series, I want to make sure we start with a good foundation. 
+Let me answer some questions you might be having.
+
+**Q:** Who are you?
+**A:** I'm Isabela, a UX/UI designer at [Quansight Labs](https://labs.quansight.org/) who 
+cares about accessibility and is fortunate to work somewhere where that is a respected concern. 
+I also spend time in the Jupyter ecosystem—especially around JupyterLab—though that is not the 
+only open source community you can find me in. I like to collect gargoyles, my hair is pink, 
+and I love the sunflower emoji :sunflower:. It's nice to meet you!
+
+**Q:** What is the Jupyter ecosystem and JupyterLab?
+**A:** [Project Jupyter](https://jupyter.org/) is an organization that produces open source software 
+and open standards. The Jupyter ecosystem is a term used to describe projects that are directly a 
+part of or support Project Jupyter. JupyterLab is a one of it’s primary projects and a staple for 
```

Comment:
For readability:

That are either a direct part of or support the Project Jupyter.